cmd/vic-machine/common/cnetworks.go

```diff
@@ -26,21 +26,50 @@ import (
 	"github.com/vmware/vic/pkg/ip"
 )
 
+type TrustLevel int
+
+const (
+	Open TrustLevel = iota
```

Comment:
I think this will make "Open" the default - can we have Published be the default please.